src/components/shared/NetworkSettings.js

```diff
@@ -46,10 +46,12 @@ class Settings extends PureComponent<Props> {
   }
 }
 
-export default explicitConnect<{||}, StateProps, DispatchProps>({
-  mapStateToProps: state => ({
-    searchString: getNetworkSearchString(state),
-  }),
-  mapDispatchToProps: { changeNetworkSearchString },
-  component: Settings,
-});
+export const NetworkSettings = explicitConnect<{||}, StateProps, DispatchProps>(
```

Comment:
If you run `yarn lint-fix-js`, it will automatically apply the prettier formatting. Many editors can also be configured to run the fixing for prettier at save time, which is how I have VS Code configured.

Reply:
Done it!! Thank you!!